git_machete/cmd.py

```diff
@@ -1636,7 +1371,7 @@ def log_result() -> Generator[str, None, None]:
                     def branch_def_to_str(lb: str, lb_or_rb: str) -> str:
                         return lb if lb == lb_or_rb else f"{lb_or_rb} (remote counterpart of {lb})"
 
-                    joined_branch_defs = ", ".join(map(tupled(branch_def_to_str), branch_defs))
+                    joined_branch_defs = ", ".join(map(utils.tupled(branch_def_to_str), branch_defs))
```

Comment:
Nit: `tupled`, `get_second` and other low-level utils can probably be `from`-imported

Reply:
Same for e.g. `flat_map`, `excluding`